tox.ini

```diff
@@ -11,6 +11,7 @@ deps = -r{toxinidir}/requirements-test.txt
 
 [testenv:lint]
 commands =
+    black --check reconcile tools e2e_tests
```

Comment:
I'd probably just add a new environment for this called `format`. You can just add the `black` dependency there explicitly to avoid needing all of the others?

Reply:
Good point.

Added a dedicated env for `format` as you suggested. Further added a dedicated `requirements-format.txt`. Moved all `requirements-*.txt` files into a `requirements/` directory.

Does that cover the review?

Reply:
I added one more inline note about `posargs` as mentioned above. I'd also double-check for any old references to requirements files in `README.md`, unless I've missed that they've already been updated?

Reply:
Very good point! Forgot to update README :see_no_evil: 

Fixed

Reply:
Looks good! I'll resolve this thread.